server/controllers/importers/sif.js

```diff
@@ -122,20 +122,23 @@ module.exports = function (sif) {
                 });
             }
         });
+  }
+
+  var network = {
+    genes: emptySifFile ? [] : genes.map(function (geneName) {
+      return { name: geneName };
+    }),
```

Comment:
Ah I didn’t realize that we get a `[" "]`. Seems like a bug in how we parse then; a lone space should produce nothing. But that can be a subsequent task then.